### ISSUE TITLE
Increase vrouter_backend_sync_timeout_ms in volumedriver config

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#1191]
We observed the following on a recent multidatacenter `fargo` environment.

When trying to perform a `move` (between 2 local nodes) of a volume with a active `write` test on it. The `move` of the volume failed every time.

After increasing the `vrouter_backend_sync_timeout_ms` parameter from 5 seconds to 60 seconds in the volumedriver config. The volume moved successfully.

The volume could not be moved because: `that means that the volume could not write out its dirty data within the specified timeout to the backend, so the transfer is aborted.`

**Comment by khenderick** [comment]
Fixed by #1192 and #1193, packaged in openvstorage-2.7.5-fargo.4 and openvstorage-2.7.6-rev.4344.a90a602

**Comment by kinvaris** [comment]
PASSED: Checked a vpool config on our CI environments
```
root@ovs-node01-1604:~ovs config get /ovs/vpools/016dfe8b-ae26-4499-afa9-b7953b1ac0ad/hosts/myvpool01EYHk1VhLIHiY0H9L/config | grep vrouter_backend_sync_timeout_ms
        "vrouter_backend_sync_timeout_ms": 60000

```
